### DISCUSSION TITLE
Is the privacy policy correct?

### DISCUSSION THREAD

**Discussion by xipe-totec-gnu** [jithware/brethap#99] [Q&A] [answered]
The program is free/libre software but I'm not a programmer so I don't know how to check this by myself but...  Does this program really collect that much information? Name, email, gender, location, pictures?!

Maybe this is just needed for legal reasons, however I'd like to know if this actually collected. Or if maybe this isn't as scary as it seems. 

Does the children privacy section means that this program is not intended for people younger than 13 years old?

Thank you very much for your answers

**Comment by jithware** [comment, marked as answer]
At the time of the creation of [PRIVACY.md](https://github.com/jithware/brethap/blob/master/PRIVACY.md), Google started requiring all apps, regardless if they collected any data or not, to have a privacy policy. Currently the [privacy policy ](https://github.com/jithware/brethap/commit/798626b39b38e0503b433a87d09573b7712c7368) is just a boiler plate template that was downloaded to cover many possible scenarios. As you stated, any flutter/dart programmer can review the source code and verify what, if any, is collected.

If you know of, or know of someone who can suggest, a freely obtained privacy policy that better reflects the current application, please let me know and I will update it.
